src/EPR.Calculator.Frontend/Constants/StaticHelpers.cs

```diff
@@ -5,7 +5,7 @@ public class StaticHelpers
         public const string FileNotSelected = "Please select a file";
         public const string FileMustBeCSV = "Your file must have the extension .csv";
         public const string FileNotExceed50KB = "The CSV must be smaller than 50KB";
-        public const string Path = @"wwwroot/assets/SchemeParameterTemplates/SchemeParameterTemplate.v0.1.xlsx";
+        public const string Path = @"wwwroot/assets/SchemeParameterTemplates/SchemeParameterTemplate.v1.1.xlsx";
```

Comment:
Can this be a configuration item?
`SchemeParameterTemplate.v1.1.xlsx` @johnbenjamin-defra @Skumarhpa

Reply:
Hi @udaydefra, if we are going to host the template file on Azure, then we can go with the configuration item, as we can avoid doing a release whenever the template file gets modified. If not, then I think constant should be fine. Thanks.